src/kw/APIConfig.cpp

```diff
@@ -0,0 +1,61 @@
+/**
+ * \file APIConfig.cpp
+ *
+ * \copyright Copyright (C) 2019 Kano Computing Ltd.
+ *            License: http://www.gnu.org/licenses/gpl-2.0.txt GNU GPL v2
+ *
+ * \brief Configuration file for web services
+ *
+ */
+
+
+#include "mercury/kw/APIConfig.h"
+
+#include <string>
+#include <utility>
+#include <vector>
+
+#include "mercury/config.h"
+
+
+using Mercury::KanoWorld::APIConfig;
+using std::move;
+using std::string;
+using std::vector;
+
+
+APIConfig::APIConfig(vector<string> config_files) :
+        API_URL_KEY("API_URL"),
+        // FIXME: Find a cross-platform mechanism for this
+        SYSTEM_PATH("/etc/mercury-api.yaml") {
+#if CMAKE_BUILD_TYPE == Debug
+    config_files.push_back(
+        string(CMAKE_PROJ_BASE_DIR) + "conf/dev.yaml");
```

Comment:
Maybe make this yaml specific to the module?
```suggestion
        string(CMAKE_PROJ_BASE_DIR) + "conf/kw_dev.yaml");
```